src/historywork/BatchDownloadVerifyBucketsWork.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright 2018 Stellar Development Foundation and contributors. Licensed
```

Comment:
this doesn't belong in this commit: split into refactor commits and changes to how we download and verify buckets

Reply:
(also new files need to be added to the Visual Studio project)